source/includes/_tipser-react-elements.md

```diff
@@ -197,58 +199,112 @@ import { Store } from '@tipser/tipser-elements';
 Note: the `Store` component is updating the top-level page URL (when it's tabs are clicked). For this reason, please double check if it won't interfere with your web framework. 
 For the same reason, it's not recommended to include more than one `<Store />` on a single page. 
 
- 
-## Customizing Tipser Elements Styles ##
+## Checkout
+Is an HTML widget displaying a whole Checkout component allowing a user to make a purchase on the POS site, using Tipser infrastructure. The Checkout component can be used in two ways: 
 
-Tipser Elements are the "building blocks" designed to fit your page as much as possible. We created the styling in a way that delivers a nice look & feel from the start, but also allows you to change them easily to fit your unique sense of style. For example, all elements' `font-family` and `font-size` attributes are set to inherit them from the host page. If you need to change some other styles, please overwrite the certain CSS classes.
+Stand-alone `Checkout` react component displayed on one site;
 
-### Product Card ###
+Modular Checkout component displaying parts of it as different react Components in POS controlled order on different subsites;
```

Comment:
I don't think we should say "POS controlled order". This acronym is not clear for external readers. Maybe we could just write:

> in a custom order?